charts/pytorchjob/templates/ingress.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.ingress -}}
+apiVersion: extensions/v1beta1
```

Comment:
versions before 1.14  can use extensions/v1beta1, version after 1.14 use "networking.k8s.io/v1beta1",did you test "extensions/v1beta1" on k8s v1.16.9?